test/utils.test.js

```diff
@@ -0,0 +1,59 @@
+// eslint-disable-next-line spaced-comment
+/* eslint-env node, jest */
+/* global BigInt */
+
+"use strict";
+
+const utils = require("../lib/output/utils");
+
+describe("utils.js", () => {
+  describe("isObject", () => {
+    const BIG_INT_PLACEHOLDER = "BigInt placeholder";
+
+    /** @type {Array<string | number | bigint | boolean | symbol | null | undefined>} */
+    const primitives = [
+      123,
+      "string",
+      Symbol.iterator,
+      true,
+      BIG_INT_PLACEHOLDER,
+      null,
+      undefined
+    ];
+
+    // eslint-disable-next-line new-cap, valid-typeof
```

Comment:
Please add `BigInt` to https://github.com/jsdom/webidl2js/blob/701f958b609c9571c95c47998fdaf4c2aee97a28/.eslintrc.json#L165 rather than using `new-cap`.

Reply:
Actually, this&nbsp;is&nbsp;unnecessary as&nbsp;of&nbsp;**ESLint&nbsp;v6.2.0**.

Turns&nbsp;out that&nbsp;I&nbsp;had an&nbsp;outdated&nbsp;**ESLint** version&nbsp;installed when&nbsp;I&nbsp;originally wrote&nbsp;this.